en/docs/reference/connectors/msazuredatalakestorage-connector/microsoft-azure-data-lake-storage-connector-reference.md

```diff
@@ -0,0 +1,1587 @@
+            <td>The operation will be performed only if the resource has been modified since the specified time.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>sourceIfNoneMatch</td>
+            <td>The operation will be performed only if the resource's ETag does not match the specified ETag value.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>destinationLeaseId</td>
+            <td>This request will succeed only if the provided leaseId matches the actual lease on the resource.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>destinationIfUnmodifiedSince</td>
+            <td>The operation will be performed only if the resource has not been modified since the specified time.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>destinationIfMatch</td>
+            <td>The operation will be performed only if the resource's ETag matches the specified ETag value.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>destinationIfModifiedSince</td>
+            <td>The operation will be performed only if the resource has been modified since the specified time.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>destinationIfNoneMatch</td>
+            <td>The operation will be performed only if the resource's ETag does not match the specified ETag value.</td>
+            <td>No</td>
+        </tr>
+    </table>
+
+    **Sample configuration**
+
+    ```xml
+    <msazuredatalakestorage.renamePath configKey="con1">
+        <fileSystemName>{${payload.fileSystemName}}</fileSystemName>
+        <directoryName>{${payload.directoryName}}</directoryName>
+        <newDirectoryName>{${payload.newDirectoryName}}</newDirectoryName>
+        <responseVariable>msazuredatalakestorage_renamePath_1</responseVariable>
+        <overwriteBody>true</overwriteBody>
+        <timeout></timeout>
+        <newFileSystemName>{${payload.newFileSystemName}}</newFileSystemName>
+        <sourceLeaseId></sourceLeaseId>
+        <sourceIfUnmodifiedSince></sourceIfUnmodifiedSince>
+        <sourceIfMatch></sourceIfMatch>
+        <sourceIfModifiedSince></sourceIfModifiedSince>
+        <sourceIfNoneMatch></sourceIfNoneMatch>
+        <destinationLeaseId></destinationLeaseId>
+        <destinationIfUnmodifiedSince></destinationIfUnmodifiedSince>
+        <destinationIfMatch></destinationIfMatch>
+        <destinationIfModifiedSince></destinationIfModifiedSince>
+        <destinationIfNoneMatch></destinationIfNoneMatch>
+    </msazuredatalakestorage.renamePath>
+    ```
+
+    **Sample request**
+
+    ```json
+    {
+        "fileSystemName": "sales",
+        "directoryName": "section1/marketing",
+        "newDirectoryName": "section1/marketing_renamed",
+        "newFileSystemName": "sales_renamed"
+    }
+    ``` 
+
+
+
+
+
+## Error codes related to Microsoft Azure Storage Connector
```

Comment:
Shouldn't this be Microsoft Azure Data Lake Storage Connector related error codes?